judo-tatami-jsl-jsl2ui/src/main/epsilon/transformations/ui/modules/structure/class.etl

```diff
@@ -1,19 +1,18 @@
 @greedy
-@primary
-@lazy
 rule ClassType
     transform s: JSL!TransferDeclaration
     to t: UI!ui::data::ClassType {
-		
+        guard: actorDeclaration.getExposedTransferObjects().includes(s)
+
         t.setId(actorDeclaration.name + "/(jsl/" + s.getId() + ")/ClassType");
 
         t.name = s.getFqName() + "::ClassType";
         //t.transferObjectTypeName = s.transferObjectTypeName();
         t.simpleName = s.name;
         //t.setPackageNameTokens(s);
 
-        t.isMapped = s.isMapped();
-        t.isOptional = s.isMapped();
+        // t.isMapped = s.isMapped();
+        // t.isOptional = s.isMapped();
```

Comment:
Why is it commented?

Reply:
Transfer member transformation will come later.